Sources/Workspace/PinsStore.swift

```diff
@@ -136,18 +128,15 @@ public final class PinsStore {
         }
         // The repo should already be pinned.
         guard let pin = pinsMap[package] else { throw PinOperationError.notPinned }
-        // Remove pin and save the state.
+        // Remove pin
```

Comment:
nit: Comments should end in punctuation. We probably don't even need this comment now.